src/Datadog.Trace.ClrProfiler.Native/CorProfiler.cpp

```diff
@@ -52,6 +53,15 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* pICorProfilerInfoUnk
     }
     */
 
+    WCHAR integration_file_path[MAX_PATH]{};
+    const DWORD integration_file_path_length = GetEnvironmentVariable(L"DATADOG_INTEGRATIONS", integration_file_path, _countof(integration_file_path));
```

Comment:
We should include this file and set the env. variable in the Windows installer so it doesn't break.